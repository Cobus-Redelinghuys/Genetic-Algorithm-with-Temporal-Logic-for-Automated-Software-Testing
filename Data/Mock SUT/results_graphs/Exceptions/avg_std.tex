Table \ref{tab:HP:GA:Exceptions:average fitness} tabulates the final average, standard deviations, and overall average for the average fitness that the different GATLAST configurations configured with different exceptions weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
exceptions & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.403 & 0.022 & 0.355\\\hline
0.25 & 0.389 & 0.03 & 0.34\\\hline
0.5 & 0.374 & 0.02 & 0.329\\\hline
0.75 & 0.378 & 0.02 & 0.329\\\hline
1.0 & 0.369 & 0.027 & 0.321\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the average fitness metric that the of different exceptions configured GATLASTs were achieved}
\label{tab:HP:GA:Exceptions:average fitness}
\end{table}