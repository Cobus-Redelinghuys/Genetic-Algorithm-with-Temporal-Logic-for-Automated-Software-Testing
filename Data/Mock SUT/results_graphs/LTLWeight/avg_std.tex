Table \ref{tab:HP:GA:LTLWeight:average fitness} tabulates the final average, standard deviations, and overall average for the average fitness that the different GATLAST configurations configured with different ω₁ weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
ω₁ & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.398 & 0.018 & 0.349\\\hline
0.25 & 0.411 & 0.033 & 0.351\\\hline
0.5 & 0.374 & 0.02 & 0.329\\\hline
0.75 & 0.381 & 0.016 & 0.328\\\hline
1.0 & 0.374 & 0.021 & 0.327\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the average fitness metric that the of different ω₁ configured GATLASTs were achieved}
\label{tab:HP:GA:LTLWeight:average fitness}
\end{table}