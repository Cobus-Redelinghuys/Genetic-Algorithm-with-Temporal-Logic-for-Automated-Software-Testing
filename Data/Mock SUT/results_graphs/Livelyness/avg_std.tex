Table \ref{tab:HP:GA:Livelyness:average fitness} tabulates the final average, standard deviations, and overall average for the average fitness that the different GATLAST configurations configured with different proper termination weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
proper termination & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.335 & 0.021 & 0.302\\\hline
0.25 & 0.379 & 0.02 & 0.326\\\hline
0.5 & 0.374 & 0.02 & 0.329\\\hline
0.75 & 0.402 & 0.028 & 0.349\\\hline
1.0 & 0.408 & 0.025 & 0.36\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the average fitness metric that the of different proper termination configured GATLASTs were achieved}
\label{tab:HP:GA:Livelyness:average fitness}
\end{table}