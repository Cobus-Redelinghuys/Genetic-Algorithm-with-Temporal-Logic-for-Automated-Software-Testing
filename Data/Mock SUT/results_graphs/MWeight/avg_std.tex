Table \ref{tab:HP:GA:MWeight:average fitness} tabulates the final average, standard deviations, and overall average for the average fitness that the different GATLAST configurations configured with different ω₂ weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
ω₂ & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.23 & 0.011 & 0.188\\\hline
0.25 & 0.327 & 0.02 & 0.272\\\hline
0.5 & 0.374 & 0.02 & 0.329\\\hline
0.75 & 0.437 & 0.027 & 0.38\\\hline
1.0 & 0.467 & 0.028 & 0.414\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the average fitness metric that the of different ω₂ configured GATLASTs were achieved}
\label{tab:HP:GA:MWeight:average fitness}
\end{table}