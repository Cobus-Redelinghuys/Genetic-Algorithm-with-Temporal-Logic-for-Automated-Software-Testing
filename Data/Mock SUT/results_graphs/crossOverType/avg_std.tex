Table \ref{tab:HP:GA:crossOverType:average fitness} tabulates the final average, standard deviations, and overall average for the average fitness that the different GATLAST configurations configured with different crossover type weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
crossover type & Final average & Final standard deviation & Overall average\\
\hline
N-point crossover & 0.377 & 0.029 & 0.333\\\hline
One point crossover & 0.374 & 0.02 & 0.329\\\hline
Segmented crossover & 0.377 & 0.011 & 0.335\\\hline
Uniform crossover & 0.341 & 0.022 & 0.317\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the average fitness metric that the of different crossover type configured GATLASTs were achieved}
\label{tab:HP:GA:crossOverType:average fitness}
\end{table}