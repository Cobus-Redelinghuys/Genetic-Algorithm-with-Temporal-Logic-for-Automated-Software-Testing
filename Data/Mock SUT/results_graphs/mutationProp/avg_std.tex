Table \ref{tab:HP:GA:mutationProp:average fitness} tabulates the final average, standard deviations, and overall average for the average fitness that the different GATLAST configurations configured with different mutation probability weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
mutation probability & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.446 & 0.026 & 0.357\\\hline
0.25 & 0.387 & 0.028 & 0.335\\\hline
0.5 & 0.374 & 0.02 & 0.329\\\hline
0.75 & 0.365 & 0.025 & 0.323\\\hline
1.0 & 0.36 & 0.02 & 0.322\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the average fitness metric that the of different mutation probability configured GATLASTs were achieved}
\label{tab:HP:GA:mutationProp:average fitness}
\end{table}