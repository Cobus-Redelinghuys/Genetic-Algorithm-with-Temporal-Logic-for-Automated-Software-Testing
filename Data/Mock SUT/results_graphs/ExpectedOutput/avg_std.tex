Table \ref{tab:HP:GA:ExpectedOutput:average fitness} tabulates the final average, standard deviations, and overall average for the average fitness that the different GATLAST configurations configured with different expected output weights were able to achieve.
\begin{table}[tbh!]
\centering
\begin{adjustbox}{trim=0cm 0cm 0cm 0cm, clip=true}
\begin{tabular}{|c|c|c|c|}
\hline
expected output & Final average & Final standard deviation & Overall average\\
\hline
0.0 & 0.357 & 0.023 & 0.309\\\hline
0.25 & 0.375 & 0.018 & 0.328\\\hline
0.5 & 0.374 & 0.02 & 0.329\\\hline
0.75 & 0.415 & 0.039 & 0.349\\\hline
1.0 & 0.416 & 0.028 & 0.351\\\hline
\end{tabular}
\end{adjustbox}
\caption{Final average, standard deviation, and overall average for the average fitness metric that the of different expected output configured GATLASTs were achieved}
\label{tab:HP:GA:ExpectedOutput:average fitness}
\end{table}